dep_checker/dependencies.py

```diff
@@ -59,6 +59,7 @@ def get_cpe(self, repo_path: Path) -> Optional[str]:
 
 dependencies_per_branch: dict[str, list[str]] = {
     "main": common_dependencies,
+    "v22.x": common_dependencies,
     "v21.x": common_dependencies,
     "v20.x": common_dependencies,
     "v18.x": common_dependencies, # ["per-branch-dependencies"],
```

Comment:
This appears to be the same for every version -- is it better to have to explicitly add new semver-majors, or have some sort of default?

Reply:
Having some sort of default sounds better to me

Reply:
I agree with the default, was surprised it did not just work since we've made it independent in other places.  A default with a warning might make sense if that is possible.